packages/wallet/src/redux/state.ts

```diff
@@ -322,3 +335,13 @@ export function getCommitments(store: SharedData, channelId: string): Commitment
 }
 
 export { NewLedgerChannel };
+
+export function isTerminalProcessProtocolState(
```

Comment:
Is `isTerminalProtocolState` a better name?

Reply:
Renamed